### ISSUE TITLE
migration merge conflict in 1.3.2 

### ISSUE THREAD

**Issue by hachacha** [BirkbeckCTP/janeway#862]
trying to get new code up on our dev server and using the 1.3.2 branch.

![image](https://user-images.githubusercontent.com/1020384/52443469-c7dbc880-2af3-11e9-83b2-effba7255506.png)

the migrations folder has 
```
0024_auto_20190129_1647.py
0024_auto_20190201_1015.py
0025_fix_bad_journal_urls_in_template.py
```
0025 has 
```dependencies = [
        ('core', '0024_auto_20190201_1015'),
    ]
```

so `0024_auto_20190129_1647` should be referenced like here: https://github.com/BirkbeckCTP/janeway/commit/a352b879df35a5c3576e5e1cc80bb93e2f308a93

however, you have something named 0025 already...

**Comment by mauromsl** [comment]
Hi @hachaha , 
Branch 1.3.2 doesn't have `0024_auto_20190129_1647.py`. This migration was added from a PR of yours, so perhaps you accidentally merged your branch into your `1.3.2`?

As you said master also has a migration prefixed `core.0025`, that conflict will be resolved once we merge 1.3.2 into master.

We have a jenkins job running migrations on every push to master, so we can detect this sort of issues quite quickly now:

https://jenkins.janeway.systems/job/janeway-master-ci/

**Comment by hachacha** [comment]
ah ha! yes that file was in my pycache folder. thank you.
